_posts/2021-01-15-galaxy-wg.md

```diff
@@ -0,0 +1,24 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'New Galaxy Community Organization: Working Groups'
+tags: [galaxy]
+---
+
+<br>
+The Galaxy community is undergoing an __organizational transformation__, with the main focus on improving our global coordination in different areas by defining [__Working Groups__](https://galaxyproject.org/community/wg/). 
+
+The Working Groups are __distributed across the globe__ and diverse in the topics covered  (admin, backend, UI/UX...). Although their initial definition is set, they might be refined over time.
```

Comment:
The working groups are not distributed but the members? The working groups are open for everyone and have members from all over the globe...